lib/streamlit/elements/multiselect.py

```diff
@@ -104,6 +104,14 @@ def _check_and_convert_to_indices(
     return [opt.index(value) for value in default_values]
 
 
+def get_default_count(default: Union[Iterable[Any], Any, None]):
+    if default is None:
+        return 0
+    if not isinstance(default, Iterable):
```

Comment:
Oh, interesting. I'm actually a bit surprised this works since I thought `Iterable` was used for type annotations only. Apparently you can actually check for most iterables like this, but I'm not sure if it works 100% of the time.

A more reliable method that I'm pretty sure works 100% of the time (even if the iterable thing wasn't written with type annotations) is to try to call the `iter` function on the object. We have an `is_iterable` helper function in `type_util.py` that already exists for this.

Reply:
Should be fixed! thanks! I forgot we have type utils!